iRODS/server/config/lockFileDir/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
```

Comment:
was this removal intended?

Reply:
Nope, must have added it by accident. 4.1's build process (or at least run-in-place) seems to touch this file every time.